lib/puppet/type/group.rb

```diff
@@ -80,7 +80,9 @@ def sync
 
     newproperty(:members, :array_matching => :all, :required_features => :manages_members) do
       desc "The members of the group. For directory services where group
```

Comment:
I'm fine with leaving it for a more comprehensive style pass through these. :+1: